src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -49,9 +49,6 @@ export class HuiCardPicker extends LitElement {
 
   protected render(): TemplateResult | void {
     return html`
-      <h3>
-        ${this.hass!.localize("ui.panel.lovelace.editor.edit_card.pick_card")}
```

Comment:
Remove the no longer used key from en.json

Reply:
I still use that key when no title for the view is set.